exchanges/bybit/bybit_websocket.go

```diff
@@ -891,10 +826,10 @@ func (by *Bybit) getPairFromCategory(category, symbol string) (currency.Pair, as
 	case "option":
 		assets = append(assets, asset.Options)
 	default:
-		return currency.EMPTYPAIR, 0, fmt.Errorf("category %q not supported for incoming symbol %q", category, symbol)
+		return currency.EMPTYPAIR, 0, fmt.Errorf("incoming symbol %q %w: %q", symbol, errUnsupportedCategory, category)
```

Comment:
🐒 This error feels back to front
```suggestion
		return currency.EMPTYPAIR, 0, fmt.Errorf("%w %q for symbol %q %w: %q", errUnsupportedCategory, category, symbol)
```